plugins/services/src/js/components/modals/ServiceActionDisabledModal.js

```diff
@@ -363,7 +363,7 @@ class ServiceActionDisabledModal extends React.Component {
   getGroupDeleteMessage() {
     const { actionID, service, intl } = this.props;
     const serviceID = service ? service.getId() : "";
-    const { selectedServices } = this.getServiceTypes(isPackage);
+    const { selectedServices } = this.getServiceTypes(isSDKService);
```

Comment:
You need to use the isPackage since we need a different delete command for packages, i.e. clean out zookeeper as well, and for other services, i.e. stuff that has been installed directly with Marathon.
I guess you saw an empty command because we added the flags artificially to a marathon app, and we are here assuming that an SDK service is installed through Cosmos. Maybe we can't assume that, but I think that is ok.